src/Microsoft.AspNet.Routing/RouteDirection.cs

```diff
@@ -3,9 +3,19 @@
 
 namespace Microsoft.AspNet.Routing
 {
+    /// <summary>
+    /// Indicates whether ASP.NET routing is processing a URL from a client or generating a URL.
+    /// </summary>
     public enum RouteDirection
     {
+        /// <summary>
+        /// A URL from a client is being processed.
+        /// </summary>
```

Comment:
Copied from MSDN. Changed to `Indicates whether ASP.NET routing is processing a URL from an HTTP request or generating a URL.`